Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -516,7 +516,7 @@ extension BrowserViewController: TabManagerDelegate {
         //previous?.webView.scrollView.delegate = nil
         selected?.webView.navigationDelegate = self
```

Comment:
It looks like you're adding selectedTab checks for navigation below, so we should move these to tab creation rather than registering/unregistering them when the tab changes.